storage/innobase/log/log0recv.cc

```diff
@@ -95,10 +95,15 @@ recv_sys_t *recv_sys = nullptr;
 otherwise.  Note that this is false while a background thread is
 rolling back incomplete transactions. */
 volatile bool recv_recovery_on;
+
+#if defined(UNIV_HOTBACKUP) || defined(XTRABACKUP)
```

Comment:
Aligning it with upstream code.

Reply:
I would say we should add this since we do not add logic behind UNIV_HOTBACKUP.

Also, here you are making `volatile lsn_t backup_redo_log_flushed_lsn;` definition only if XTRABACKUP macro is true. Later we use this variable at `recv_parse_or_apply_log_rec_body` inside a `#ifndef UNIV_HOTBACKUP
` macro block, which is invalid.

Reply:
fixed. moved volatile lsn_t backup_redo_log_flushed_lsn where it was previously.